ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/maintenance.j2

```diff
@@ -0,0 +1,52 @@
+{% if maintenance is arista.avd.defined %}
+
+## Maintenance
+
+### Maintenance defaults
+
+Default maintenance bgp profile: **{{ maintenance.default_bgp_profile | arista.avd.default('Default') }}**
+
+Default maintenance interface profile: **{{ maintenance.default_interface_profile | arista.avd.default('Default') }}**
+
+Default maintenance unit profile: **{{ maintenance.default_unit_profile | arista.avd.default('Default') }}**
+
+### Maintenance profiles
+
+| BGP profile | Initiator route-map
+| -------- | ----- |
+{%     for bgp_profile in maintenance.bgp_profiles | arista.avd.natural_sort %}
+| {{ bgp_profile }} | {{ maintenance.bgp_profiles[bgp_profile].initiator.route_map_inout }} |
+{%     endfor %}
+
+| Interface profile | Rate monitoring load interval | Rate monitoring threshold in/out |
+|-------------------|-------------------------------|------------------|
+{%     for interface_profile in maintenance.interface_profiles | arista.avd.natural_sort %}
+| {{ interface_profile -}}
+| {{ maintenance.interface_profiles[interface_profile].rate_monitoring.load_interval | arista.avd.default(60) -}}
+| {{ maintenance.interface_profiles[interface_profile].rate_monitoring.threshold | arista.avd.default(100) }} |
+{%     endfor %}
+
+| Unit profile | on-boot duration |
+| -------- | ----- |
+{%     for unit_profile in maintenance.unit_profiles | arista.avd.natural_sort %}
+| {{ unit_profile }} |{{ maintenance.unit_profiles[unit_profile].on_boot.duration | arista.avd.default('disabled') }} |
+{%     endfor %}
+
+### Maintenance units
+
+| Unit | Interface groups | BGP groups | Unit profile | Quiesce |
+|------|------------------|------------|--------------|---------|
+{%     for unit in maintenance.units | arista.avd.natural_sort %}
+| {{ unit -}}
+| {% if maintenance.units[unit].groups.interface_groups is arista.avd.defined %}{{ maintenance.units[unit].groups.interface_groups | arista.avd.natural_sort | join(', ') }}{% else %}-{% endif %}
+| {% if maintenance.units[unit].groups.bgp_groups is arista.avd.defined %}{{ maintenance.units[unit].groups.bgp_groups | arista.avd.natural_sort | join(', ') -}}{% else %}-{% endif %}
+| {% if maintenance.units[unit].profile is arista.avd.defined %}{{ maintenance.units[unit].profile }}{% else %}{{ maintenance.default_unit_profile | arista.avd.default('Default') }}{% endif -%}
```

Comment:
Our `arista.avd.default` filter can take multiple values, so this can be simplified to `maintenance.units[unit].profile | arista.avd.default(maintenance.default_unit_profile, 'Default')`